src/libLookup/Lookup.cpp

```diff
@@ -3311,7 +3341,8 @@ void Lookup::CommitTxBlocks(const vector<TxBlock>& txBlocks) {
       if (!m_currDSExpired) {
```

Comment:
should do `RejoinNetwork` if `GetDSInfo` returns `false`?

Reply:
Done